addons/website_event_track/views/event_track_templates_agenda.xml

```diff
@@ -74,83 +77,94 @@
         </div>
-        <hr class="mt-2 mb-2"/>
 
         <t t-set="locations" t-value="locations_by_days[day]"/>
         <!-- Day Agenda -->
-        <div class="o_we_online_agenda">
-        <table id="table_search" class="table table-sm border-0 h-100">
-            <!--Header-->
-            <tr>
-                <th class="border-0 position-sticky"/>
-                <t t-foreach="locations" t-as="location">
-                    <th t-if="location" class="active text-center">
-                        <span t-out="location and location.name or 'Unknown'"/>
-                    </th>
-                </t>
-            </tr>
-
-            <!-- Time Slots -->
-            <t t-set="used_cells" t-value="[]"/>
-            <t t-foreach="time_slots[day]" t-as="time_slot">
-                <t t-set="is_round_hour" t-value="time_slot == time_slot.replace(minute=0)"/>
-                <t t-set="is_half_hour" t-value="time_slot == time_slot.replace(minute=30)"/>
-
-                <tr t-att-class="'%s' % ('active' if is_round_hour else '')">
-                    <td class="active">
-                        <b t-if="is_round_hour" t-out="time_slots[day][time_slot]['formatted_time']"/>
-                    </td>
-
-                    <t t-foreach="locations" t-as="location">
-                        <t t-set="tracks" t-value="time_slots[day][time_slot].get(location, {})"/>
-                        <t t-if="tracks">
-                            <t t-foreach="tracks" t-as="track">
-                                <t t-set="_classes"
-                                    t-value="'text-center %s %s %s' % (
-                                        'event_color_%s' % (track.color or 0),
-                                        'event_track h-100' if track else '',
-                                        'o_location_size_%d' % len(locations),
-                                    )"/>
-                                <t t-if="track.location_id and track.location_id == location">
-                                    <td t-att-rowspan="tracks[track]['rowspan']"
-                                        t-att-class="_classes">
-                                        <t t-call="website_event_track.agenda_main_track"/>
-                                    </td>
+        <div t-attf-class="{{event_has_complex_day and 'container-fluid' or 'container-xl'}} pe-0">
+            <div class="o_we_online_agenda mb-3">
+                <table id="table_search" class="table table-sm border-0 h-100">
+                    <!--Header-->
+                    <tr>
+                        <th class="border-0 position-sticky"/>
+                        <t t-foreach="locations" t-as="location">
+                            <th t-if="location" class="o_we_online_agenda_header_location active text-center">
+                                <span t-out="location and location.name or 'Unknown'"/>
```

Comment:
I think if you just have it as part of an expression it won't be translatable
I guess that's how it was before but would be better to have it in a separate `t-set` while we're here